src/renderer/Widget/AppSpaceListWidget/components/PreviewContent.tsx

```diff
@@ -6,6 +6,14 @@ const IconType = ({type}: {type: string}) => {
 };
 
 const PreviewContent = ({contentList}: {contentList: ModalAppSpaceContent[]}) => {
+  function redirectApp(categoryType: string, id: number) {
+    let path = `/#/space/${id}`;
+
+    if (categoryType === 'app') {
+      path = `${id}`;
+    }
+    window.location.replace(`${window.location.origin}/k/${path}`);
```

Comment:
Use `window.location.href` to navigate. `window.location.replace` will not add a new record to the history.